.github/workflows/dev_version_script.py

```diff
@@ -11,74 +11,102 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+import re
 import argparse
 from pathlib import Path
-import importlib
 
-import re
+try:
+    import semver
+except ImportError as exc:
+    raise ImportError("Unable to import semver. Install semver by running `pip install semver`") from exc
 
-VERSION_FILE_PATH = "pennylane_lightning/core/_version.py"
+DEV_PRERELEASE_TAG_PREFIX = "dev"
+DEV_PRERELEASE_TAG_START = "dev1"
```

Comment:
The dev-version in previous cycles started with dev0.

```suggestion
DEV_PRERELEASE_TAG_START = "dev0"
```